modules/robin/CHANGELOG.md

```diff
@@ -0,0 +1,13 @@
+# Changelog
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
+and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
+
+## Unreleased
+
+### Added
+* Made `TextZones.Zone` type public
+* Made `Words.WordScope` type public
+
+## 8.0.8
```

Comment:
This isn't valid. Either add the date and entries for 8.0.8 or just remove this please.

- [ ] Fix invalid changelog